### PR TITLE
Add bugfixes to changelog [skip ci]

### PR DESCRIPTION
**Pull request by palango**
Add some bug fixes that weren't mentioned before.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,8 +2,10 @@
 Changelog
 =========
 
-* :bug:`1261` REST API now returns json error for invalid endpoints
-* :feature:`1230` Unless specifically provided gas price and gas limit are now dynamically calculated from the `eth_gasPrice()` and latest block's limit respectively.
+* :bug:`1217` Correctly decode network events in the REST API.
+* :bug:`1224` Fix internal server error on REST endpoint ``/events/tokens/`` for a non-existing tokens.
```
Comment (LefterisJP) [inline docs/changelog.rst]: Remove `a`

File: docs/changelog.rst [hunk 2]
```diff
@@ -2,13 +2,16 @@
 Changelog
 =========
 
-* :bug:`1261` REST API now returns json error for invalid endpoints
-* :feature:`1230` Unless specifically provided gas price and gas limit are now dynamically calculated from the `eth_gasPrice()` and latest block's limit respectively.
+* :release:`0.3.0 <2018-02-21>`
+* :bug:`1217` Correctly decode network events in the REST API.
+* :bug:`1224` Fix internal server error on REST endpoint ``/events/tokens/`` for non-existing tokens.
+* :bug:`1261` REST API now returns json error for invalid endpoints.
+* :feature:`1230` Unless specifically provided gas price and gas limit are now dynamically calculated from the ``eth_gasPrice()`` and latest blocks limit respectively.
```
Comment (hackaugusto) [inline docs/changelog.rst]: this looks like the wrong issue number

Comment (hackaugusto) [reply docs/changelog.rst]: wasn't it #1473

Comment (LefterisJP) [reply docs/changelog.rst]: 1473 was the internal sort of issue that explains to us what we want to do (iow use web3), while I guess 1230 was the first occurence of gas price being too low causing problems which eventually leads us to the solution.


But that's only a guess. This is almost a year ago.

**Pull request merged**